### DISCUSSION TITLE
How do I set the initial size of a node?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [bcakmakoglu/vue-flow#1353] [Q&A] [answered]
I'd like to set the initial view to have nodes in specific positions and sizes. Positions is clear enough, but I can't seem to find anything related to the size besides `NodeResizer`.

**Comment by bcakmakoglu** [comment, marked as answer]
You can pass `width` and `height` to a node, which will be applied as a style.

```ts
interface Node {
  // ...

  /**
   * Fixed width of node, applied as style
   * You can pass a number which will be used in pixel values (width: 300 -> width: 300px)
   * or pass a string with units (width: `10rem` -> width: 10rem)
   */
  width?: number | string | WidthFunc
  /**
   * Fixed height of node, applied as style
   * You can pass a number which will be used in pixel values (height: 300 -> height: 300px)
   * or pass a string with units (height: `10rem` -> height: 10rem)
   */
  height?: number | string | HeightFunc
```

**Reply by CodyBontecou** [reply]
Perfect, that does it.

**Reply by CodyBontecou** [reply]
Ah, actually, maybe not. 

Is it possible to set the initial width + height and allow adjustments after the fact?

**Reply by bcakmakoglu** [reply]
Sure you can update these values whenever you want and it should be reflected on the node reactively
